test/test_apps_timers_unit.js

```diff
@@ -0,0 +1,452 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of ThingEngine
+//
+// Copyright 2016-2018 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Giovanni Campagna <gcampagn@cs.stanford.edu>
```

Comment:
Please update the header here and in the other file with your name, and use this year for the copyright line.